test/functional/cmdLineTests/jfr/src/org/openj9/test/WorkLoad.java

```diff
@@ -35,6 +35,10 @@ public class WorkLoad {
 	public static double average;
 	public static double stdDev;
 
+	private static long globalCounter = 0;
+	static interface GlobalLoack {};
```

Comment:
Looks like a typo, but it doesn't block the PR from being merged. It can be addressed separately since the main PR has already been merged.